### DISCUSSION TITLE
Question about importing .nvblx file to curobo

### DISCUSSION THREAD

**Discussion by Sepirence** [NVlabs/curobo#133] [Software Q&A] [answered]
I am getting familiar with using curobo, and now, I am trying to input my own nvblox map data to the curobo.
Starts with your example code `/isaac-sim/collision_checker.py`, I imported my own .nvblx file (generated with isaac-ros-nvblox) and .obj file. (generated with isaac-ros-nvblox and translated with MeshLab) 
Mesh is successfully loaded on isaac simulator, but cannot work well with collision checking. 

However, I could not open .nvblx file to check whether it is generated well or not. Do you have any suggestion for checking collision block or some option that you used when generating .nvblx file?

**Comment by balakumar-s** [comment, marked as answer]
Is it possible for you to share a .nvblx file? We need the esdflayer from nvblox.

**Reply by Sepirence** [reply]
https://drive.google.com/file/d/16ruW7DVdPUCjYt3BxOSla0QA4oeobPfI/view?usp=drive_link

I uploaded my google drive. Do you need .obj file, too?

+ I opened the data with sqlite viewer, and I find that esdf_layer data only have 0 z-axis. I made this data with nvblox save_map service call, but is there any another way to generate 3D esdf data?

**Reply by Sepirence** [reply]
I made it. it was problem about Pose expression. I set the quarternions all zero, and it makes problem.
Your work is so delicate. :+1:

**Reply by balakumar-s** [reply]
Glad you figured it out
